frontend/src/employee-frontend/components/vasu/components/MultiSelectQuestion.tsx

```diff
@@ -54,9 +65,14 @@ export function MultiSelectQuestion({
         <ValueOrNoRecord
           text={options
             .filter((option) => selectedValues.includes(option.key))
-            .map((o) => o.name)
+            .map((o) =>
+              textValue && textValue[o.key]
+                ? `${o.name} (${textValue[o.key]})`
```

Comment:
At least in the current use case, the emphasis is a bit backwards here in my opinion. The "text value" is more important than the option's "name" (eg. "muualle, minne?"). I know it's problematic to omit the "name" completely because this implementation is so generic that it might be important later. I don't know what to replace it with either, maybe `${o.name}: ${textValue[o.key]}` or something.

I'm not necessarily suggesting that you should change the model, but did you think about the option of modeling the solutions as an "other" option with a text field that could be optionally included at the end of a `MultiSelectQuestion`?